sts_auth/cli.py

```diff
@@ -44,7 +44,7 @@ def cli():
                   'Proceed with caution and use a tool like `pass` to securely store your secrets.'
               )
               )
-@click.option('--console', '-c', is_flag=True,
+@click.option('--browser', '-b', is_flag=True,
```

Comment:
The `console` parameter to `authenticate` (line 58) needs to be updated to `browser`.